packages/medusa/src/services/cart.ts

```diff
@@ -2604,6 +2604,7 @@ class CartService extends TransactionBaseService {
       .withTransaction(transactionManager)
```

Comment:
question/thought: How do we delete custom adjustments on a cart? Should we expose the same config as with OrderEdits?

Reply:
The purpose of the config on OrderEdits `refreshAdjustments` is to reuse the method internally and prevent the deletion of custom adjustments in some instances since we are deleting old and recomputing adjustments (which is not the case for carts). We could add similar logic for carts but I don't know if there is a case where we would need to delete custom adjustments (since we are not creating them), wdyt?

Since custom adjustments are created by a user with some custom logic by using the `LineItemAdjustemntService` directly, should the user be responsible for the cleanup as well?